### PR TITLE
Allow loading of gzipped 3d tiles in server.js

### PR DESCRIPTION
**Pull request by shehzan10**
Ref https://github.com/AnalyticalGraphicsInc/3d-tiles-samples/pull/19 and https://github.com/AnalyticalGraphicsInc/3d-tiles-samples/pull/11#issuecomment-307770995

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -39,6 +42,7 @@
     var mime = express.static.mime;
     mime.define({
         'application/json' : ['czml', 'json', 'geojson', 'topojson'],
+        'application/octet-stream' : ['b3dm', 'i3dm', 'pnts', 'cmpt'],
```
Comment (emackey) [inline server.js]: Please add these to web.config also, thanks!

Comment (shehzan10) [inline server.js]: Oh actually these were added to master in the 3D Tiles PR.
However, my adding them made a duplicate copy here (I was working on master). I'll remove it.

**Comment by shehzan10** [comment]
bump

**Comment by mramato** [comment]
Thanks.

**Pull request merged**